Source/JavaScriptCore/bytecode/CodeBlock.cpp

```diff
@@ -1730,6 +1745,12 @@ StructureStubInfo* CodeBlock::findStubInfo(CodeOrigin codeOrigin)
             if (stubInfo->codeOrigin == codeOrigin)
                 return stubInfo;
         }
+        if (auto* jitData = dfgJITData()) {
+            for (auto& stubInfo : jitData->stubInfos()) {
+                if (stubInfo.codeOrigin == codeOrigin)
+                    return &stubInfo;
+            }
```

Comment:
Because it is used for linked DFG / FTL code. And because DFG / FTL code requires live pointer to `StructureStubInfo*` when emitting a code right now, so it needs to be `Bag<StructureStubInfo>`.